kubectl-plugin/src/main.rs

```diff
@@ -24,6 +29,9 @@ struct CliArgs {
     /// The operation to be performed.
     #[structopt(subcommand)]
     operations: Operations,
+    /// Output Format
+    #[structopt(default_value = "", short, long)]
+    output: String,
```

Comment:
what do we get with --help? Do we need to add more comments above with the possible values?

Reply:
```
kubectl-mayastor 0.1.0
The types of operations that are supported

USAGE:
    kubectl-mayastor [OPTIONS] <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -o, --output <output>    Output Format [default: ]
    -r, --rest <rest>        Rest endpoint

SUBCOMMANDS:
    get      'Get' resources
    help     Prints this message or the help of the given subcommand(s)
    scale    'Scale' resources
```

Currently, it shows something like this. I would add the supported formats here as well.

Reply:
So, the default shows up as "empty". This is where adding the enum would perhaps make it clearer as it'd show the default value of the enum.
For the possible values you can actually add the `possible_values` option to the structopt argument.

Reply:
Got your point. I was not aware of these functionalities of `StructOpts`. Changed output to `enum`. Added `possible_values`. Removed those redundant checks. The `--help` looks like this now. Thanks.

```
[nix-shell:~/github/Abhinandan-Purkait/mayastor-control-plane]$ ./target/debug/kubectl-mayastor --help                                                    
kubectl-mayastor 0.1.0
The types of operations that are supported

USAGE:
    kubectl-mayastor [OPTIONS] <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -o, --output <output>    The Output, viz yaml, json [default: none]  [possible values: yaml, json, none]
    -r, --rest <rest>        The rest endpoint, parsed from KUBECONFIG, if left empty 

SUBCOMMANDS:
    get      'Get' resources
    help     Prints this message or the help of the given subcommand(s)
    scale    'Scale' resources
```